src/app.js

```diff
@@ -1 +1,12 @@
-'use strict';
+const fs = require('fs');
+
+(() => {
+  const [source, dest] = process.argv.slice(2);
+
+  try {
+    fs.copyFileSync(source, dest);
+  } catch (err) {
+    // eslint-disable-next-line
+    console.error(err);
```

Comment:
While logging the error is useful, you might also want to provide a more user-friendly message indicating that the file copy operation failed. This can help users understand what went wrong without needing to interpret the error object.